config/development.yml

```diff
@@ -1,4 +1,22 @@
-webAppHost: "http://localhost:8080"
+webAppHost: 'http://localhost:8080'
+
+jwt:
+  token: 'JWT_TOKEN'
+
+communication:
+  services: ['twilio', 'sendgrid']
+
+twilio:
+  verify:
+    accountSid: 'AC.TWILIO_ACCOUNT_SID'
+    authToken: 'TWILIO_AUTH_TOKEN'
+  messaging:
+    messagingServiceSid: 'TWILIO_MESSAGING_SERVICE_SID'
+
+sendgridApiKey: 'SG.SENDGRID_API_KEY'
```

Comment:
@rayescmata  - Any reason why this was added? The app should work without these integrations and thus not requiring us to define any configuration values.